### ISSUE TITLE
Menu entry to send group to top/bottom

### ISSUE THREAD

**Issue by alphapapa** [josh-berry/tab-stash#436]
### Problem Statement

Sometimes I need to move a group to the top or bottom of the list (or a parent group), and in a large group or list of groups, that can mean significant dragging-and-scrolling, taking several seconds to get to the right place.

### Preferred Solution(s)

I'd like to be able to right-click a group and select "Move to top/bottom" to have the group moved to the top or bottom of the list or parent group.

### Alternative Solution(s)

A somewhat related idea, which could help solve the problem to some extent is, https://github.com/josh-berry/tab-stash/issues/124.  But this feature would still be useful on its own.

### Additional Context

This extension is just indispensable for me.  I used Firefox for so long without it (since before it was called Firefox).  Thank you for your work on it.

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
Makes sense, thanks for the feature request! And sorry for the delayed response—still catching up after the holidays.

And thanks for the kind words, glad to hear it's helpful. Nice to hear from you again!
